src/wml_exception.cpp

```diff
@@ -93,3 +93,11 @@ std::string missing_mandatory_wml_key(
 			"mandatory key '$key|' isn't set.", symbols);
 	}
 }
+
+std::string missing_mandatory_wml_tag(
+		  const std::string &section
+		, const std::string &tag)
+{
+	// TODO in 1.19: revisit this when we can add new strings
+	return missing_mandatory_wml_key(section, tag);
```

Comment:
If it's redirecting to `missing_mandatory_wml_key`, should it maybe add square brackets around the tag name? I get that this is only temporary, but…

"Mandatory key '[state_disabled]' isn't set"

seems better than

"Mandatory key 'state_disabled' isn't set"

Reply:
Agreed, done in the latest push.